packages/xstate-wallet/src/workflows/approve-budget-and-fund.ts

```diff
@@ -95,9 +113,22 @@ export const approveBudgetAndFundWorkflow = (
     updateBudget: (context: WorkflowContext, event) => {
       return store.updateOrCreateBudget(context.budget);
     },
-    createAndFundLedger: (context: WorkflowContext, event) => {
-      // TODO: Hook up to workflow when it exists
-      return Promise.resolve();
+    createAndFundLedger: ({player, hub, budget}: WorkflowContext) => {
+      // TODO: Abstract into function that takes a budget and returns a outcome
+      const ethBudget = budget.budgets[ETH_ASSET_HOLDER_ADDRESS];
+      const playerItem: AllocationItem = {
+        destination: player.destination,
+        amount: ethBudget.pending.playerAmount
+      };
+      const hubItem: AllocationItem = {
+        destination: player.destination,
```

Comment:
Shouldn't this be the hub's destination?

Reply:
Yup! That should be fixed